### DISCUSSION TITLE
Customizing the colorScheme prop beyond adding a new color to the theme?

### DISCUSSION THREAD

**Discussion by nvsd** [chakra-ui/chakra-ui#3559] [Q&A]
I was wondering if I could customize the way that colorScheme determines the colors to use based on light/dark themes?

**Comment by dodas** [comment]
Hey @nvsd!
The shades of `colorScheme` an component uses is usually defined within it's `variant`.
For example, shades for Button's default variant – `solid`, are defined [here](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/button.ts#L103).
You would need to override theme for each component where you want to change it. It is not really globalized/streamlined in any way.

**Reply by nvsd** [reply]
That's perfect! Thanks!
